env/elastica/composer.json

```diff
@@ -13,14 +13,14 @@
     ],
     "require": {
         "php": "^7.0",
-        "phpdocumentor/phpdocumentor":"~2.8",
-        "fabpot/php-cs-fixer":"1.11.*",
-        "mayflower/php-codebrowser":"~1.1",
-        "pdepend/pdepend":"~2.2",
-        "phploc/phploc":"~2.1",
-        "phpmd/phpmd":"~2.3",
-        "phpunit/phpunit":"~5.6",
-        "sebastian/phpcpd":"~2.0",
-        "squizlabs/php_codesniffer":"~2.5"
+        "phpdocumentor/phpdocumentor": "~2.8",
+        "friendsofphp/php-cs-fixer": "^2.0",
```

Comment:
Could you have a look at CI? I assume some more updates are needed: https://travis-ci.org/ruflin/Elastica/jobs/467233662

Reply:
Ok! I'll check it ASAP